src/d/actor/d_a_player_main.cpp

```diff
@@ -5082,7 +7607,343 @@ BOOL daPy_lk_c::checkAttentionPosAngle(fopAc_ac_c* actor, cXyz** pOutPos) {
+        } else if (!checkHeavyStateOn() && (m3730.abs2XZ() > 25.0f)) {
+            local_e8.set(current.pos.x - 100.0f * m3730.x, 120.0f + current.pos.y,
+                         current.pos.z - 100.0f * m3730.z);
+        } else {
+            local_e8.set(current.pos.x - (100.0f * m3644) * cM_ssin(m3640), 120.0f + current.pos.y,
+                         current.pos.z - (100.0f * m3644) * cM_scos(m3640));
+        }
+        local_128 = &local_e8;
+        bVar5 = true;
+    } else if ((pfVar12 != NULL) &&
+               (cLib_distanceAngleS(cLib_targetAngleY(&current.pos, &pfVar12->eyePos), m34DE) <=
+                0x6000))
+    {
+        local_128 = &(pfVar12->eyePos);
+        bVar5 = true;
+    } else if (checkModeFlg(ModeFlg_00000080 | ModeFlg_08000000)) {
+        if ((!dComIfGp_event_runCheck()) || (mDemo.getDemoMode() == daPy_demo_c::DEMO_UNK39_e)) {
+            if (dComIfGp_getAttention().getLookTarget() != NULL) {
+                local_128 = &dComIfGp_getAttention().getLookTarget()->eyePos;
+                bVar5 = true;
+            } else {
+                if ((((checkAttentionPosAngle(pfVar14, &local_128)) ||
+                      (checkAttentionPosAngle(mpAttnActorAction, &local_128))) ||
+                     (checkAttentionPosAngle(mpAttnActorX, &local_128))) ||
+                    ((checkAttentionPosAngle(mpAttnActorY, &local_128) ||
+                      (checkAttentionPosAngle(mpAttnActorZ, &local_128)))))
+                {
+                    bVar5 = true;
+                } else {
+                    if (dComIfGp_getDetect().chk_attention(&local_e8) &&
+                        (cLib_distanceAngleS(cLib_targetAngleY(&current.pos, &local_e8), m34DE) <=
+                         0x6000))
+                    {
+                        local_128 = &local_e8;
+                        bVar5 = true;
+                    } else if (m34C3 == 10) {
+                        local_e8.set(current.pos.x - 10.0f * (m3730.x + m36B8.x),
+                                     120.0f + current.pos.y,
+                                     current.pos.z - 10.0f * (m3730.z + m36B8.z));
+                        if (cLib_distanceAngleS(cLib_targetAngleY(&current.pos, &local_e8),
+                                                m34DE) <= 0x6000)
+                        {
+                            local_128 = &local_e8;
+                            bVar5 = true;
+                        }
+                    }
+                }
+            }
+        } else if (mCurProc == daPyProc_BOTTLE_OPEN_e) {
+            pfVar12 = mActorKeepRope.getActor();
+            if (pfVar12 != NULL) {
+                local_128 = &(pfVar12->eyePos);
+                bVar5 = true;
+            }
+        } else if ((((mCurProc == daPyProc_DEMO_LOOK_WAIT_e) ||
+                     (mDemo.getDemoMode() == daPy_demo_c::DEMO_UNK04_e)) ||
+                    ((mDemo.getDemoMode() - 1 <= 1 ||
```

Comment:
Looks like a compiler optimization, this might work
```suggestion
                    ((mDemo.getDemoMode() == daPy_demo_c::DEMO_UNK01_e ||
                    mDemo.getDemoMode() == daPy_demo_c::DEMO_UNK02_e ||
```

Reply:
Doesn't work.